test/onnx_import.cpp

```diff
@@ -358,3 +358,96 @@ TEST(onnx, model_matmul)
     auto result_vectors = execute(function, inputs, "INTERPRETER");
     EXPECT_TRUE(test::all_close_f(expected_output, result_vectors.front()));
 }
+
+TEST(onnx, model_sub)
+{
+    auto function = ngraph::onnx_import::import_onnx_function(
+        ngraph::file_util::path_join(SERIALIZED_ZOO, "onnx/sub.onnx"));
+
+    Inputs inputs;
+    inputs.emplace_back(
+        ngraph::test::NDArray<float, 3>(
+            {{{1, 2, 3, 4, 5}, {6, 7, 8, 9, 10}, {11, 12, 13, 14, 15}, {16, 17, 18, 19, 20}},
```

Comment:
Please use smaller input data for testing. There is no need to make tests so verbose.